package.json

```diff
@@ -92,4 +92,4 @@
     "url": "https://github.com/tiagosiebler/binance/issues"
   },
   "homepage": "https://github.com/tiagosiebler/binance#readme"
-}
\ No newline at end of file
+}
```

Comment:
Language server did this again. Hope you don't mind.

Reply:
This is fine, as long as eslint is happy - otherwise one of the build steps will fail